portal/server/helpers/getDealIdsCheckersApprovalAmendments.js

```diff
@@ -0,0 +1,20 @@
+const { PORTAL_AMENDMENT_STATUS } = require('@ukef/dtfs2-common');
+const api = require('../api');
+
+/**
+ * Retrieves deal IDs that have amendments status ready for checker's approval.
+ * @param {string} userToken - A valid user token.
+ * @returns {Promise<Array<string>>} A promise that resolves to an array of deal IDs with amendments in progress.
+ */
+const getDealIdsCheckersApprovalAmendments = async (userToken) => {
```

Comment:
```suggestion
const getCheckersApprovalAmendmentDealId = async (userToken) => {
```

Just thinking might be a better name, though not a blocker.

Reply:
I changed it to  getCheckersApprovalAmendmentDealIds because is an array of dealIds